src/components/words/SearchForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function SearchForm({ term, onChange }) {
+  return (
+    <>
```

Comment:
```suggestion
    <label>
    Search:
    <input name="searchTerm" value={term} onChange={onChange} />
    </label>
```
Good opportunity to use label tags!